src/components/card/cart-product-card/cartProductCard.css

```diff
@@ -1,54 +1,53 @@
-.card-footer{
-    margin-top:auto;
+.card-footer {
+    margin-top: auto;
```

Comment:
margin-top: doesn't work. It takes margin:0 in auto case. Please check if its working or not if not then you can remove this peice of code

Reply:
Yes margin-top auto works. You can check it on my hosted site : https://rapidstore-dev.netlify.app/